assets/sass/components/setup/_googlesitekit-setup.scss

```diff
@@ -127,14 +128,21 @@
 	margin-right: 16px;
 }
 
-.googlesitekit-wizard {
+.googlesitekit-setup__warning {
+	align-items: center;
+	display: flex;
+
+	svg {
+		width: 34px;
+	}
```

Comment:
@asvinb Is this necessary? It seems like this should already be the case from the component here: https://github.com/google/site-kit-wp/blob/160b5d1393cee4cce60574827564b5b02cd8b78e/assets/js/components/legacy-notifications/warning.js#L28

Reply:
Yes it's necessary @aaemnnosttv , to prevent the SVG from shrinking as we resize the browser. Since we are using flex, everything will remain on the same line